openreview/conference/templates/reviewProcess.js

```diff
@@ -14,41 +12,50 @@ function(){
+          message: "A submission to " + SHORT_PHRASE + ", for which you are an official area chair, has received a review. \n\nTitle: " + note.content.title + "\n\nComment: " + note.content.review + "\n\nTo view the review, click here: " + baseUrl + "/forum?id=" + note.forum + '&noteId=' + note.id
         };
         promises.push(or3client.or3request( or3client.mailUrl, areachair_mail, 'POST', token ));
       }
 
       if (note.readers.includes('everyone') || note.readers.includes(PAPER_REVIEWERS)) {
         var reviewer_mail = {
-          "groups": [PAPER_REVIEWERS],
-          "subject": "[" + SHORT_PHRASE + "] Review posted to your assigned paper: \"" + forum.content.title + "\"",
-          "message": "A submission to " + SHORT_PHRASE + ", for which you are a reviewer, has received an official review. \n\nTitle: " + note.content.title + "\n\nComment: " + note.content.review + "\n\nTo view the review, click here: " + baseUrl + "/forum?id=" + note.forum + '&noteId=' + note.id
+          groups: [PAPER_REVIEWERS],
+          ignoreGroups: [note.tauthor],
+          subject: "[" + SHORT_PHRASE + "] Review posted to your assigned paper: \"" + forum.content.title + "\"",
+          message: "A submission to " + SHORT_PHRASE + ", for which you are a reviewer, has received a review. \n\nTitle: " + note.content.title + "\n\nComment: " + note.content.review + "\n\nTo view the review, click here: " + baseUrl + "/forum?id=" + note.forum + '&noteId=' + note.id
         };
         promises.push(or3client.or3request( or3client.mailUrl, reviewer_mail, 'POST', token ));
       }
 
       if (note.readers.includes('everyone') || note.readers.includes(PAPER_AUTHORS)) {
         var author_mail = {
-          "groups": [PAPER_AUTHORS],
-          "subject": "[" + SHORT_PHRASE + "] Review posted to your submission: \"" + forum.content.title + "\"",
-          "message": "Your submission to " + SHORT_PHRASE + " has received an official review. \n\nTitle: " + note.content.title + "\n\nComment: " + note.content.review + "\n\nTo view the review, click here: " + baseUrl + "/forum?id=" + note.forum + '&noteId=' + note.id
+          groups: [PAPER_AUTHORS],
+          subject: "[" + SHORT_PHRASE + "] Review posted to your submission: \"" + forum.content.title + "\"",
+          message: "Your submission to " + SHORT_PHRASE + " has received a review. \n\nTitle: " + note.content.title + "\n\nComment: " + note.content.review + "\n\nTo view the review, click here: " + baseUrl + "/forum?id=" + note.forum + '&noteId=' + note.id
         };
         promises.push(or3client.or3request( or3client.mailUrl, author_mail, 'POST', token ));
       }
 
-      promises.push(or3client.addGroupMember(REVIEWERS_SUBMITTED, note.signatures[0], token));
-
-      return Promise.all(promises);
+      return Promise.all(promises)
+      .then(function(result){
+        return or3client.addGroupMember(REVIEWERS_SUBMITTED, note.signatures[0], token);
```

Comment:
I had to remove the submitted reviewer notification because this group can be empty and the /messages endpoint throws an error when the group is empty.